packages/plugin-rust-cargo/index.js

```diff
@@ -0,0 +1,45 @@
+import liveResolverQuery from '@octolinker/resolver-live-query';
+import * as TOML from '@iarna/toml';
+import insertLink from '@octolinker/helper-insert-link';
+import { tomlRegExKeyValue } from '@octolinker/helper-regex-builder';
+
+function linkDependency(blob, key, value) {
+  const regex = tomlRegExKeyValue(key, value);
+  return insertLink(blob, regex, this);
+}
+
+function processTOML(blob, plugin, config) {
+  let results = [];
+  const json = TOML.parse(blob.toString());
+  const toml = TOML.stringify(json.dependencies).split('\n');
```

Comment:
What's the reason to convert the result of `TOML.parse` to a string again? `TOML.parse` returns a JSON object so you can iterate over it like this

```js
Object.entries(json.dependencies).forEach(([name, version]) => {
  console.log(`${name} @ ${version}`);
});
```

Reply:
This allows you to simplify this PR even further and remove `tomlRegexBuilder`

Reply:
> What's the reason to convert the result of `TOML.parse` to a string again?

I was copying, pasting, and modifying until things started to work 😄

Reply:
That's how everything begins 😆